tool/tsh/kube.go

```diff
@@ -959,6 +963,10 @@ func updateKubeConfig(cf *CLIConf, tc *client.TeleportClient, path string) error
 
 	values, err := buildKubeConfigUpdate(cf, kubeStatus)
 	if err != nil {
+		if trace.IsNotFound(err) {
+			log.Debugf("Skipping kubernetes config update: %v", err)
+			return nil
+		}
```

Comment:
Instead of returning NotFound from buildKubeConfigUpdate and handling it here, why not just check `if (kubeStatus.kubeClusters) == 0` above right after `fetchKubeStatus` and return right there? Cause if there are no registered kube clusters, we shouldn't do anything anyway?

Reply:
How will the user be able to use Kube access in this case if there were no registered clusters at the time of the login? Would they need to provide --kube-cluster flag to the tsh login command or something?

If so, I think that clearer logic would be to check that --kube-cluster is not set explicity and len(kubeClusters) == 0 above, and return in that case. And add a comment as it's a pretty confusing logic.